### PR TITLE
(1) UR Selection

### PR DESCRIPTION
**Pull request by BenRJG**
Added parameter to select which arm to use in rviz sim

### CODE REVIEW

File: README.md
Comment (miguelprada) [inline README.md:84]: This should probably not go into this repository. Furthermore, if you change it, we can already replace the reference altogether with the soon to be default repository location, i.e. https://gitlab.com/tracebot/tracebot_mockup.git.

File: README.md
Comment (miguelprada) [inline README.md:39]: Maybe add a short comment in the like of: _Robot arm model used for the left arm. Must be one of: (ur3, ur3e, ..., ur16e)_?

**Review by miguelprada — CHANGES_REQUESTED** [review]
Other than the two minor comments, I think this looks good to be merged.

**Review by BenRJG — COMMENTED** [review]
Made the changes to the readme :)

**Review by miguelprada — APPROVED** [review]

**Comment by miguelprada** [comment]
Will wait for the CI run to finish before merging, to be on the safe side.

**Comment by BenRJG** [comment]
> Will wait for the CI run to finish before merging, to be on the safe side.

No problem. I'm continuing work with the camera however I'm not going to pull request this until everything's together, as I'll pull and merge first to make sure everything's right :)

**Pull request merged**
